### ISSUE TITLE
Alternate ID Acquisition Method

### ISSUE THREAD

**Issue by someONEgood** [fitzchivalrik/compass#11]
I inadvertently noticed when using QoL Bars that when using it's built in icon select, it basically shows you the entire icon library complete with IDs. I cross-checked this with the recent housing icons I looked up via xlapi ai to make sure. The only difference is it needs a 0 added to the start of the ID. 

For instance the open Mansion icon is 060756 and appears as 60756 in QoL Bars Icon Browser:
![060756](https://user-images.githubusercontent.com/46905309/137594957-887d2fd4-68f3-41f7-9070-2b4f739c10ec.JPG)

The trade-off being you have to wade through a wall of icons to find what you're looking for of course, but for less tech savvy (despite your great instructions) it might be helpful.

**Comment by fitzchivalrik** [comment]
Yes, thank you! I used the icon browser of the QOLBar plugin during development (and still use it) for verifying IDs and discover adjacent ones. It is not always straightforward though to make out which icons are used on the map and which are not. I always try to test each ID in the wild in various locations, too.  
Getting wild IDs is easier when going through the `_NaviMap`, too, I believe.

> The only difference is it needs a 0 added to the start of the ID.

You do not really need the 0 there, the ID is treated as a number anyway and 010 is the same as 10 ^^.

I leave this open as it does not hurt to use the icon browser if one has QOLBar anyway to confirm/check IDs, I did not add that to the Readme though as I did not want to add a 'dependency' to QOLBar.
